Command/HealthCheckCommand.php

```diff
@@ -28,6 +29,12 @@ protected function configure()
                     InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
                     'Additional reporters to run.',
                     array()
+                ),
+                new InputOption(
```

Comment:
that's a new feature, it makes sure the cli command outputs error codes suitable for nagios (exitCode 0 : ok, 1: got warnings, 2: got criticals)